test/resources/systemd.resource

```diff
@@ -52,7 +52,7 @@ Systemctl    # robocop: disable=too-long-keyword
         ...    sudo=True
         ...    return_stdout=True
         ${log_text}=    Execute Command
-        ...    journalctl -u ${unit_name} -o short | tail -n 100
+        ...    journalctl -u ${unit_name} -o short | tail -c 32000
```

Comment:
Maybe we could write the output to a file instead of truncating it?

Reply:
This command includes the output in the RF log. The idea was to limit the size of the buffer to 32K, which is anyway the max size displayed in the RF log. If we redirect it to a file, it won't be in the log.
Maybe, I'm missing something.